diesel/src/macros/macros_from_codegen.rs

```diff
@@ -72,6 +72,18 @@ macro_rules! infer_table_from_schema {
 ///
 /// This macro can only be used in combination with the `diesel_codegen` or
 /// `diesel_codegen_syntex` crates. It will not work on its own.
+///
+/// Example:
+/// ```
```

Comment:
Please add an empty line above code blocks, and below headlines. Bonus points for adding lang annotations to all code blocks, even though `rust` is the default (just my personal preference) :)